### DISCUSSION TITLE
Expand Examples Please

### DISCUSSION THREAD

**Discussion by mmk-embarkvet** [cdklabs/cdk-monitoring-constructs#222] [General]
Nothing to see here, sorry :)

**Comment by ghost** [comment]
Would some of [the tests](https://github.com/cdklabs/cdk-monitoring-constructs/blob/main/test/monitoring/aws-ecs-patterns/FargateServiceMonitoring.test.ts) be sufficient as examples, or are you looking for something in particular?

**Reply by mmk-embarkvet** [reply]
They were, apologies for not getting back for so long. Thanks!

**Comment by DesAWSume** [comment]
Would be good to have some example of using some of the core method you guys wrote.
